source/code/plugins/operation_lib.rb

```diff
@@ -39,9 +39,27 @@ def filter(record, time)
 
       return {}
     end
+
+    def filter_generic(record,time)
+      if record.is_a?(Hash) && !record.empty? && record.has_key?("message")
+        return {
+          "Timestamp"=>OMS::Common.format_time(time),
+          "OperationStatus"=>"Warning",
+          "Computer"=>`hostname`.strip,
+          "Detail"=>record["message"]
+        }
+      end
+      return {}
+    end
  
-    def filter_and_wrap(record, time)
-      data_item = filter(record, time)
+    def filter_and_wrap(tag, record, time)
+      case tag
+      when "oms.operation.buffer"
```

Comment:
You should not determine the operation by checking the tag name. It will block the plugin from reuse.